src/bindings/typescript/serializers.ts

```diff
@@ -738,6 +738,8 @@ const map: Serializer<MapSerializerDescr> = {
                     src.valueFormat === undefined
                 ) {
                     value = { kind: "uint", bits: 256 };
+                } else if (src.valueFormat === "coins") {
+                    value = { kind: "uint", bits: 124 };
```

Comment:
Shouldn't this be changed in the `src/storage/allocator.ts` too?

Reply:
@novusnota Which place exactly do you mean?

Reply:
Oh, nevermind, realised that's redundant :)